### DISCUSSION TITLE
Question 81: Phenology tab PP_IDs not populating on import	[WG?]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1159] [Q&A]
Description: If I add phenology data by import, PP_IDs aren’t generated. In the screenshot below, I imported 3 fields into one row and PP_ID stayed blank. When I added a second row (“Gravid”) by hand, PP_IDs were generated for both rows. Is this a problem? Do I need to designate PP_IDs during import, or …. ?

![81a](https://github.com/user-attachments/assets/78184e0d-9500-4d56-91ff-7c2132014a76)
![81b](https://github.com/user-attachments/assets/872fe79f-c963-4e0c-a524-f3cac5652bc0)

**Comment by rondlg** [comment]
Response: 
[Wyatt Gaswick](mailto:wgaswick@fieldmuseum.org)– Could you share a copy of the import CSV with helpdesk?  
(IT may need to open a ticket w/ Axiell & this may relate to which fields are [or aren’t] imported in a given batch)

![81c](https://github.com/user-attachments/assets/7d40914d-7237-433c-92a1-e1a67eccedba)

**Comment by rondlg** [comment]
@magpiedin @fmjjones 

If/When you open the hornbill ticket please add it here.

**Reply by magpiedin** [reply]
Hornbill ticket added -- [IN00226864](https://customer.hornbill.com/axiell/servicemanager/request/view/IN00226864)

**Comment by rondlg** [comment]
https://fieldmuseum.zendesk.com/agent/tickets/73267
